internal/consensus/ticker.go

```diff
@@ -71,7 +71,11 @@ func (t *timeoutTicker) Chan() <-chan timeoutInfo {
 // The timeoutRoutine is always available to read from tickChan, so this won't block.
 // The scheduling may fail if the timeoutRoutine has already scheduled a timeout for a later height/round/step.
 func (t *timeoutTicker) ScheduleTimeout(ti timeoutInfo) {
-	t.tickChan <- ti
+	select {
```

Comment:
What triggered this change? Did you encounter a case where Comet blocks here for a prolonged period?